tests/test-link-preview.md

```diff
@@ -4,20 +4,74 @@ Link previews should embed previews of the contents of a hyperlink from a messag
 
 Post location variation: 
 
-1. Post as message in center channel with RHS closed (link preview should render under message) 
-2. Post as message in center channel with RHS open (link preview should render under message) 
+1. Post as message in center channel with RHS closed (preview of first link should render under message) 
+2. Post as message in center channel with RHS open (preview of first link should render under message) 
 3. Post as comment in RHS (link preview should not render) 
-4. View comment in center channel with RHS closed (link preview should render under message) 
-5. View comment in center channel with RHS open (link preview should render under message) 
+4. View comment in center channel with RHS closed (preview of first link should render under message) 
+5. View comment in center channel with RHS open (preview of first link should render under message) 
 6. Search for post in RHS with link
 
-Post the following links one per message in the above variations: 
+Test the above variations with the below sample messages (e.g. 1-A, 2-B, 3-C, 4-D, etc.) 
```

Comment:
Propose: `Test the above variations with each of the below sample messages (e.g. ...`

Do you mean 1-A, 1-B, 1-C, ... 2-A, 2-B, 2-C, etc?